shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/helper/ios/UITestLogout.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.avishek.phong.kmmic.helper.ios
+
+import co.nimblehq.avishek.phong.kmmic.data.local.datasource.TokenLocalDataSource
+import io.ktor.util.logging.KtorSimpleLogger
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+object UITestLogout: KoinComponent {
```

Comment:
What is this class for?

Reply:
It is for clearing the keychain access when running the UI test. If I don't clean up the keychain access, the test for the login screen won't pass.